lib/rules/max-top-level-suites.js

```diff
@@ -1,28 +1,27 @@
 'use strict';
 
 /**
- * @fileoverview Disallow multiple top-level suites in a single file
+ * @fileoverview Limit the number of top-level suites in a single file
  * @author Alexander Afanasyev
  */
 
-var defaultSuiteNames = [ 'describe', 'context', 'suite' ];
+var astUtil = require('../util/ast'),
+    defaultSuiteLimit = 1;
 
 module.exports = function (context) {
     var stack = [],
         topLevelDescribes = [],
-        suiteNames = context.options[0] ? context.options[0] : defaultSuiteNames;
+        suiteLimit = context.options[0] ? context.options[0] : defaultSuiteLimit;
```

Comment:
You could do `context.options[0] || defaultSuiteLimit` (shorter) or `context.options[0] !== undefined ? context.options[0] : defaultSuiteLimit` (maybe a bit more correct, if the user wants 0 suites for some reason)